inc/saiport.h

```diff
@@ -478,6 +486,15 @@ typedef enum _sai_port_attr_t
     SAI_PORT_ATTR_ADVERTISED_FEC_MODE,
 
     /**
+     * @brief Advertised port FEC AN request bit.  True means FEC is requested for 10/40/100G
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_ADVERTISED_FEC_MODE_REQUEST,
```

Comment:
correct if me if i am wrong

SAI_PORT_FEC_MODE_RS -> F2 = 1
SAI_PORT_FEC_MODE_FC -> F3 = 1 and F0 = 1

Can we just add a new FEC_MODE instead of adding a NEW attribute? 
SAI_PORT_FED_MODE_10G_FEC_REQUEST -> F1 = 1

I think F1 is only for 10G specifically, so we probably want to make it to 10G specficially.